src/opt.rs

```diff
@@ -852,6 +852,24 @@ pub enum FindSubcommands {
 
 #[derive(Debug, StructOpt)]
 pub enum MintSubcommands {
+    /// Mint a normal SPL Token from the Token Program
+    Fungible {
```

Comment:
What do you think about changing this to `Tokens` to clarify and maybe the docs comment can be:

/// Mint SPL tokens from an existing mint account using the Token Program.

Reply:
I was thinking of following the already created structure in create
metaboss create fungible
metaboss mint fungible

Would be a bit relatable for users.

Reply:
Yeah, I think the problem is that the naming is already inconsistent as "mint" basically means "create" when used for an asset like a NFT but I also have the "create" commands for setting up a fungible mint. I guess `Fungible` is ok if we change the doc comment to explain.